composer.json

```diff
@@ -31,9 +31,10 @@
     "prefer-stable": true,
     "require": {
         "php": "^8.1|^8.2",
+        "justbetter/statamic-glide-directive": "^2.1",
         "rapidez/blade-directives": "^0.6",
         "rapidez/core": "^2.13",
-        "justbetter/statamic-glide-directive": "^2.1",
+        "rapidez/sitemap": "^1.0",
```

Comment:
This should be raised when https://github.com/rapidez/sitemap/pull/2 is merged and released.